app/views/course/groups/edit.html.slim

```diff
@@ -3,6 +3,18 @@ h2= t('.header')
 = simple_form_for [current_course, @group] do |f|
   = f.input :name
 
-  = f.association :users, collection:  current_course.users
+  table.table
+    thead
+      tr
+        th = t('.name')
+        th = t('.role')
+        th
+    tbody
+      = f.simple_fields_for :group_users do |group_user_form|
+        = render 'group_user_fields', f: group_user_form
+
+  .links
+    = link_to_add_association t('.add_user'), f, :group_users,
+                              selector: 'tbody', insert_by: 'append'
```

Comment:
On hindsight, selector alone seems weird. We should allow both:
```ruby
link_to_add_association('yadda', f, :group_users, find_using: 'next', selector: 'tbody', insert_using: 'append')
```

and

```ruby
link_to_add_association('yadda', f, :group_users, find_selector: 'tbody', insert_using: 'append')
```

Also note it's `find_using` not `find_by`.